### ISSUE TITLE
[Tooltip] DEBUG_STYLE not defined on types

### ISSUE THREAD

**Issue by raunofreiberg** [reach/reach-ui#345]
Getting this error when using `DEBUG_STYLE` in `useTooltip` hook:

```
ERROR in [at-loader] ./src/components/Tooltip/Tooltip.tsx:47:53
    TS2345: Argument of type '{ DEBUG_STYLE: boolean | undefined; }' is not assignable to parameter of type 'HTMLProps<any>'.
  Object literal may only specify known properties, and 'DEBUG_STYLE' does not exist in type 'HTMLProps<any>'.
```

**Comment by chancestrickland** [comment]
Should be fixed in 0.5.4

**Comment by raunofreiberg** [comment]
Thanks! Just wanted to say that I really appreciate the swift responses and recent work that you guys have been ramping up on Reach UI :-)
